Добавил базовый перевод доки (part 2) (#241)
feature-sliced/documentation: website/i18n/en/docusaurus-plugin-content-docs/current/about/mission.md

# Mission

Here we describe the goals and limitations of the applicability of the methodology-which we are guided by when developing the methodology

- We see our goal as a balance between ideology and simplicity
- We won't be able to make a silver bullet that fits everyone

> **Nevertheless, I would like the methodology to be close and accessible to a fairly wide range of developers**

Reviewer: Мб как-то так?
```suggestion
**Nevertheless, the methodology should be close and accessible to a fairly wide range of developers**
```

## Goals

### Intuitive clarity for a wide range of developers

> The methodology should be available - for most of the team in projects

Reviewer: Кажется, более подходящее слово
```suggestion
The methodology should be accessible - for most of the team in projects
```

> *Because even with all the future tools , it will not be enough for only experienced seniors/leads to understand the methodology*

Reviewer: Текущая формулировка звучит странновато, но мб тут я и не прав и более прошаренные англичане меня поправят
```suggestion
*Because even with all the future tools , it will not be enough, if only experienced seniors/leads will understand the methodology*
```

### Solving everyday problems

The methodology should set out the reasons and solutions to our everyday problems when developing projects

**And also-attach tools to all this (cli, linters)**

> So that developers can use a *proven experience* approach that allows them to bypass long-standing problems of architecture and development

Reviewer: Кажется, лучше так
```suggestion
So that developers can use a *battle-tested* approach that allows them to bypass long-standing problems of architecture and development
```
